src/getDOM.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-unused-vars */
+import { fetch } from './fetch.js';
+
+const scoreBoard = document.querySelector('.score-container');
+export const getDOM = () => {
+  const refresh = document.querySelector('.refresh-button');
+  const addScore = document.querySelector('.add-score');
+};
+
```

Comment:
- [ ] Enable linter here. `getDOM()` seems to be a redundant function anyways. I committed it out and removed its importation in `index.js` and your app worked just fine. If you wish to have access to refresh and addScore you may have to define that in the index.js file instead.